config/routes.rb

```diff
@@ -13,6 +15,7 @@
   post :incoming, to: 'incoming#create'
 
   devise_for :users
+  resources :users, only: [:show]
```

Comment:
Isn't this a duplicate of line 3?  Run `rake routes` to see all of your generated routes.  Personally I like this syntax better but both should have the same result